### PR TITLE
update at-testset docs for verbose option

### PR DESCRIPTION
**Pull request by rfourquet**
This option was added in #33755.

### CODE REVIEW

**Comment by ronisbr** [comment]
Thanks! I really forgot the docs when I created that PR.

File: stdlib/Test/docs/src/index.md [hunk 1]
```diff
@@ -134,7 +134,8 @@ Foo Tests     |    8      8
 ```
 
 In the event that a nested test set has no failures, as happened here, it will be hidden in the
-summary. If we do have a test failure, only the details for the failed test sets will be shown:
+summary, unless the `verbose=true` option is passed.
```
Comment (ronisbr) [inline stdlib/Test/docs/src/index.md:137]: Can you please add here the following example:

```julia
julia> @testset verbose = true "Foo Tests" begin
           @testset "Animals" begin
               @test foo("cat") == 9
               @test foo("dog") == foo("cat")
           end
           @testset "Arrays $i" for i in 1:3
               @test foo(zeros(i)) == i^2
               @test foo(fill(1.0, i)) == i^2
           end
       end;
Test Summary: | Pass  Total
Foo Tests     |    8      8
  Animals     |    2      2
  Arrays 1    |    2      2
  Arrays 2    |    2      2
  Arrays 3    |    2      2
```

**Review by ronisbr — CHANGES_REQUESTED** [review]

File: stdlib/Test/docs/src/index.md (hunk 1 above)
Comment (rfourquet) [reply stdlib/Test/docs/src/index.md]: Good idea! It's added now.

**Review by ronisbr — APPROVED** [review]
Nice! LGTM.

**Comment by rfourquet** [comment]
There is one unrelated failure, in `LinearAlgebra`: https://build.julialang.org/#/builders/52/builds/5874/steps/5/logs/stdio. Maybe worth a look by someone knowing this code (@andreasnoack).

**Comment by rfourquet** [comment]
@ronisbr thanks for your review, it helps to click merge! (although I can't really do it blindly in this case ;-) )

**Comment by ronisbr** [comment]
> @ronisbr thanks for your review, it helps to click merge! (although I can't really do it blindly in this case ;-) )

Sorry! I do not have write access. I "approve" because everything you wrote about `verbose` was right :)

**Comment by rfourquet** [comment]
> Sorry! I do not have write access.

No, I meant your approval helped me to click merge! (otherwise I would have simply waited for someone else to merge)

**Comment by ronisbr** [comment]
Oh nice! Sorry again 😅

**Pull request merged**